### PR TITLE
wdl_runner: use Broad-published version of the Docker image.

### PR DESCRIPTION
**Pull request by mbookman**
Remove wdl_pipeline_from_git.yaml as it is no longer of value.
Marked the setup.sh file as deprecated, but left it for now in case there are existing users of wdl_pipeline_from_git.yaml.
Updated README files to reflect changes to "gcloud docker" syntax.

### CODE REVIEW

**Comment by mbookman** [comment]
@vdauwera - FYI

File: wdl_runner/README.md [hunk 1]
```diff
@@ -108,37 +102,28 @@ via the following commands:
...
 
 * Replace `YOUR-PROJECT-ID` with your project ID.
 
 ## (2) Run the sample workflow in the cloud
 
-### Run the pipeline, pulling code from github
-
-#### Run the following command:
+The file [./workflows/wdl_pipeline.yaml](./workflows/wdl_pipeline.yaml)
+defines a pipeline for runing WDL workflows. By default, it uses the
```
Comment (mbookman) [inline wdl_runner/README.md]: Fixed. Thanks.

File: fastqc/README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Scratch directory:
 In this example, we push the container to [Google Container Registry](https://cloud.google.com/container-registry/) via the following commands:
 ```
 docker tag ${USER}/fastqc gcr.io/YOUR-PROJECT-ID/fastqc
-gcloud docker push gcr.io/YOUR-PROJECT-ID/fastqc
+gcloud docker -- push gcr.io/YOUR-PROJECT-ID/fastqc
```
Comment (vdauwera) [inline fastqc/README.md]: Interesting, GATK4 uses this convention to separate spark arguments from the rest. Previously I didn't know this was a thing.

**Comment by vdauwera** [comment]
LGTM

**Comment by jbingham** [comment]
LGTM

**Pull request merged**